west.yml

```diff
@@ -209,7 +209,8 @@ manifest:
       path: modules/lib/picolibc
       revision: 2097f26ce7dee36922b22ce048c09cd073ebae3d
     - name: segger
-      revision: e2ff2200556e8a8f962921444275c04971a2bb3d
+      url: https://github.com/giansta/segger.git
```

Comment:
Please remove this, you need to send your change to the Zephyr repo instead

Reply:
Removed that commit. I expect I have to update `west.yml` only once that the segger repo PR  #17 is merged, right?